drivers/firmware/scmi/nxp/cpu.c

```diff
@@ -45,3 +45,39 @@ int scmi_cpu_sleep_mode_set(struct scmi_cpu_sleep_mode_config *cfg)
 
 	return 0;
 }
+
+int scmi_cpu_pd_lpm_set(struct scmi_cpu_pd_lpm_config *cfg)
+{
+	struct scmi_protocol *proto = &SCMI_PROTOCOL_NAME(SCMI_PROTOCOL_CPU_DOMAIN);
```

Comment:
What is LPM please at list reference this with the full name in the commit message.